### ISSUE TITLE
Don't stamp cosmoz-dialog template on attached

### ISSUE THREAD

**Issue by megheaiulian** [Neovici/cosmoz-dialog-old#9]
In `cosmoz-dialog-behavior` on attach it calls _createDialog which stamps a template.
That operation is expensive and when cosmoz-dialog is used in large tree attached is called synchronuosly and locks main thread.

``` js
attached: function () {
  if (this._paperDialog === null) {
   this._paperDialog = this._createDialog();
  }
}
```

**Comment by megheaiulian** [comment]
@nomego Let me know what you think about this.

**Comment by nomego** [comment]
What do I think about what? I'm sure you're right :) Do you have any proposed solution? Stamp on open? Debounce stamp?

**Comment by megheaiulian** [comment]
Actually both. Debounce or requestIddleCallback than on open if we haven't managed to create it yet cancel debounce or iddleCallback and create it .

**Comment by nomego** [comment]
Sounds good! @plequang ?

**Comment by plequang** [comment]
Sure 👍

**Comment by megheaiulian** [comment]
Closed by #10
